srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRuleValidator.java

```diff
@@ -0,0 +1,64 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import gov.va.med.srcalc.domain.model.AbstractVariable;
+import gov.va.med.srcalc.util.ValidationCodes;
+import gov.va.med.srcalc.util.ValidationUtils2;
+
+import org.springframework.expression.ParseException;
+import org.springframework.expression.spel.standard.SpelExpressionParser;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+public class EditRuleValidator implements Validator
+{
+
+    @Override
+    public boolean supports(Class<?> clazz)
+    {
+        // TODO: Investigate if this should be changed to equals since we 
+        // most likely will not have any subclasses of EditRule
+        return clazz.isAssignableFrom(EditRule.class);
+    }
+
+    @Override
+    public void validate(final Object target, final Errors errors)
+    {
+        final EditRule editRule = (EditRule) target;
+        
+        // Validate displayName constraints. See
+        // AbstractVariable.setDisplayName() because this display name obeys the
+        // same constraints.
+        ValidationUtils.rejectIfEmpty(errors, "displayName", ValidationCodes.NO_VALUE);
+        ValidationUtils2.rejectIfTooLong(errors, "displayName", AbstractVariable.DISPLAY_NAME_MAX);
```

Comment:
These references to `AbstractVariable` needlessly increase "encumbrance" of this validator. You should copy these constants into Rule (or create a single place to hold these constants since we seem to have standardized).